docs: Improving readme

perlajarillo opened the pull request:
In this PR we are including in the Readme the installation process, development notes, contributors and  contributors guidelines.

Related to #44

perlajarillo committed:
docs: Improving readme

perlajarillo committed:
docs: Include missing instalation steps
There have been included 2 more steps in the instalation process.

Fa-v commented on README.md:
```diff
@@ -1,41 +1,85 @@
+### 2. Create a project in Firebase
+
+In order to try this app you may create a new project in the Firebase Console. You can create a free account (here) [https://console.firebase.google.com/]
+
```
The parenthesis and the square brackets are mixed up it has to be like this: `You can create a free account [here] (https://console.firebase.google.com/)` 😄

perlajarillo replied:
Oh 😄! this one was trying to escape! Fixed it! 👍

Fa-v commented on README.md:
```diff
@@ -1,41 +1,85 @@
+### 5. Set the environment variables
+
+Once you have created your project in Firebase, create manually the file .env.production in the root of your project. Type the keys generated to connect to your database as follow:
 
```
Shall we highlight the files's name? `.env.production`
Do we need the `.env.dev` or just the production one?

perlajarillo replied:
I think collaborators will need just .env.production, but as a good practice, we can recommend both. I made the changes. 😉

Fa-v reviewed (changes requested):
I think this makes a great README.
Since we're are opening the possibility for contributions, do you think is a good idea to specify an OS license?

perlajarillo commented:
> I think this makes a great README.
> Since we're are opening the possibility for contributions, do you think is a good idea to specify an OS license?

I agree with you. [MIT](https://opensource.org/licenses/MIT) would be a good fit for us, what do you think?

perlajarillo committed:
doc: correct information/styles
-Correcting mistakes in markdown styles
-Include missing information (.env.dev)
-Highlight file names

Required changes in PR revision

Fa-v commented:
The MIT license is perfect! :) Sorry for the delay in my response.

perlajarillo commented:
Thanks Fabiola, don't worry about it. Let's merge this PR! 😄
